critiquebrainz/frontend/templates/navbar.html

```diff
@@ -18,33 +18,66 @@
       <ul class="nav navbar-nav">
         {% if not error %} {# Not showing this stuff on error pages. If attempt to load user info fails there, page will not render. #}
           {% if not current_user or current_user.is_anonymous %}
-            <li><a href="{{ url_for('login.index', next=request.url) }}">{{ _('Sign in') }}</a></li>
+            <li><a href="{{ url_for('login.index', next=request.url) }}">
+              <span class="glyphicon glyphicon-log-in"></span>{{ _('Sign in') }}</a></li>
           {% else %}
             <li class="dropdown">
               <a class="dropdown-toggle" data-toggle="dropdown" href="#">
                 {{ current_user.display_name }} <span class="caret"></span>
               </a>
               <ul class="dropdown-menu" role="menu">
-                <li><a href="{{ url_for('user.reviews', user_id=current_user.id) }}">{{ _('Profile') }}</a></li>
-                <li><a href="{{ url_for('profile_applications.index') }}">{{ _('Applications') }}</a></li>
+                <li>
+                  <a href="{{ url_for('user.reviews', user_id=current_user.id) }}">
+                    <span class="glyphicon glyphicon-user"></span>{{ _('Profile') }}
+                  </a>
+                </li>
+                <li>
+                  <a href="{{ url_for('profile_applications.index') }}">
+                    <span class="glyphicon glyphicon-file"></span>{{ _('Applications') }}
```

Comment:
Is there a reason you added these glyphicons in this PR? They don't line up for me and I think it makes stuff a bit cluttered.

![2019-03-19-231018_1920x1080_scrot](https://user-images.githubusercontent.com/1837631/54628907-7af8f380-4a9c-11e9-85f2-8f49612c7fa7.png)

Reply:
Let's go back to how it was.

Reply:
@paramsingh Actually, I've added changes to less file. Please recheck this after running gulp to generate updated styles.

Reply:
Should look something like this:

![add-glyphicon](https://user-images.githubusercontent.com/16593502/54672210-99530380-4b1d-11e9-95d4-9ebf235daac1.png)

Reply:
Notice that the `Statistics` box on the right side is now aligned with the `Welcome` box on the left.

Reply:
Thanks for fixing the alignment! Still it looked cleaner to me without the glyphs, any chance we can go back to how it was?